Added stars to mark favorite items (#2102)

Aga-C opened the pull request:
Hi,

I've added marking favorite items with stars, so it's visible which items from a particular album are added to favorites. I've added it in both grid view and list view in the top-right corner. In case of grid view, I had to move video duration to the bottom-right corner of the thumbnail.

![image](https://user-images.githubusercontent.com/85929121/140657744-42cef8ac-bcd3-45eb-bce7-0ef7037b39fc.png)

tibbi commented:
try adding more padding to the star, looks too big. I just added a new string mark_favorite_items, please add a new toggle into the app settings -> File thumbnail style for this pull request

tibbi commented:
guess it can be enabled by default, just test it properly with other thumbail items like file type, video duration, selection check etc, to make sure they dont overlap

Aga-C commented:
I've added a setting, made star smaller, and also moved everything on a thumbnail around to not overlap anything.

![image](https://user-images.githubusercontent.com/85929121/144069447-7574e49c-a705-436b-8a28-34b06bd3821b.png)

tibbi commented on app/src/main/res/layout/photo_item_grid.xml:
```diff
@@ -17,7 +17,7 @@
         android:layout_height="wrap_content"
         android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
+        android:layout_alignParentBottom="true"
```
you cant just change something like this, it now fully overlaps with the file name

Aga-C commented:
Should I move everything above the file name if it's visible?

tibbi commented:
no, that would hide the image then. Keep icons at the top, not bottom. Lets move the video duration to the right of the Video icon and Favorite at the top right corner. Make the Video and Favorite icons the same size and margins though so they are symmetric.

Aga-C commented:
I've changed it.

![image](https://user-images.githubusercontent.com/85929121/144234602-67442882-519f-4021-b26b-4ff84e10990e.png)

tibbi commented:
ye should be good, Ill just remove the new dimens values, I want to avoid adding useles new values like that. Ill do that in master though, it is easier to edit multiple files like that. Thanks
